platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/impl/SystemPropertiesAdmin.java

```diff
@@ -234,7 +257,20 @@ public void writeSystemProperties(Map<String, String> updatedSystemProperties) {
                 LOGGER.warn("Unable to write system user attribute file for hostname update.", e);
             }
         }
+    }
 
+    /**
+     * Overwrite attributes with those from GuestClaimsHandlerExt so system high is set.
+     */
+    private void addGuestClaimsProfileAttributes(Map<String, Object> json) {
+        Map<String, Object> selectedProfileAttributes =
+                guestClaimsHandlerExt.getSelectedClaimsProfileAttributes();
```

Comment:
Are there two different values that should be in play? One for system-high and one for guest-system-high? I didn't think there was. 

Currently we're just copying the attributes from the claims profile to the system user. This means that system high is being set to whatever the selected value was on setup. It can be changed manually at a later time but will require a system restart.